### PR TITLE
Add CRUD for passkey identity and passkey authenticator

### PR DESCRIPTION
**Pull request by louischan-oursky**
ref #2226

### CODE REVIEW

File: pkg/lib/authn/authenticator/service/conversion.go [hunk 1]
```diff
@@ -28,22 +31,55 @@ func passwordFromAuthenticatorInfo(a *authenticator.Info) *password.Authenticato
 		UserID:       a.UserID,
 		CreatedAt:    a.CreatedAt,
 		UpdatedAt:    a.UpdatedAt,
-		PasswordHash: []byte(a.Secret),
+		PasswordHash: a.Claims[authenticator.AuthenticatorClaimPasswordPasswordHash].([]byte),
```
Comment (carmenlau) [inline pkg/lib/authn/authenticator/service/conversion.go:34]: When I tried to signup with password, I got this error `interface conversion: interface {} is string, not []uint8`.

Full error logs:
```
ERRO[0063] panic occurred                                app=accounts error="interface conversion: interface {} is string, not []uint8" logger=webapp-panic-middleware stack="[log.PanicValue at /Users/carmenlau/work/authgear/authgear-server/pkg/util/log/panic.go:56 webapp.(*PanicMiddleware).Handle.func1.3 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/panic_middleware.go:53 runtime.gopanic at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/runtime/panic.go:838 db.(*HookHandle).WithTx.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/infra/db/hook_handle.go:78 runtime.gopanic at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/runtime/panic.go:838 runtime.panicdottypeE at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/runtime/iface.go:262 service.passwordFromAuthenticatorInfo at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/authn/authenticator/service/conversion.go:34 service.(*Service).Create at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/authn/authenticator/service/service.go:330 facade.(*Coordinator).AuthenticatorCreate at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/facade/coordinator.go:226 facade.AuthenticatorFacade.Create at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/facade/authenticator.go:32 nodes.(*NodeDoCreateAuthenticator).GetEffects.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/interaction/nodes/do_create_authenticator.go:38 interaction.EffectRun.apply at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/interaction/effect.go:16 interaction.(*Graph).Apply at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/interaction/graph.go:320 webapp.(*Service2).Get.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/service2.go:87 interaction.(*Service).DryRun at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/interaction/service.go:88 webapp.(*Service2).Get at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/service2.go:86 webapp.(*Controller).InteractionGet at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/controller.go:270 webapp.(*SetupRecoveryCodeHandler).ServeHTTP.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/setup_recovery_code.go:78 db.(*HookHandle).WithTx at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/infra/db/hook_handle.go:89 webapp.(*Controller).Serve at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/controller.go:143 webapp.(*SetupRecoveryCodeHandler).ServeHTTP at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/setup_recovery_code.go:120 deps.(*RootProvider).Handler.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:170 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*DynamicCSPMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/dynamic_csp_middleware.go:49 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 web.StaticSecurityHeadersMiddleware.Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/web/static_security_headers_middleware.go:10 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 csrf.(*csrf).ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/packages/pkg/mod/github.com/gorilla/csrf@v1.7.1/csrf.go:307 webapp.(*CSRFMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/csrf_middleware.go:52 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*TutorialMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/tutorial_middleware.go:26 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*ClientIDMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/client_id.go:34 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*WeChatRedirectURIMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/wechat_redirect_uri_middleware.go:73 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*ColorSchemeMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/x_color_scheme.go:78 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*UILocalesMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/ui_locales.go:48 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*SessionMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/session_middleware.go:39 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.IntlMiddleware.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/intl_middleware.go:14 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 httputil.NoCache.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/util/httputil/no_cache.go:12 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 session.(*Middleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/session/middleware.go:64 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*PanicMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/panic_middleware.go:64 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*PublicOriginMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/public_origin_middleware.go:36 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RequestMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/middleware_request.go:39 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 middleware.(*SentryMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/infra/middleware/sentry.go:22 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 middleware.(*BodyLimitMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/infra/middleware/body_limit.go:16 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 middleware.(*PanicMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/infra/middleware/panic.go:63 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 httprouter.(*Router).Handler.func1 at /Users/carmenlau/.asdf/installs/golang/1.18.2/packages/pkg/mod/github.com/julienschmidt/httprouter@v1.3.0/router.go:275 httprouter.(*Router).ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/packages/pkg/mod/github.com/julienschmidt/httprouter@v1.3.0/router.go:387 httproute.(*Router).ServeHTTP at /Users/carmenlau/work/authgear/authgear-server/pkg/util/httproute/httproute.go:82 http.serverHandler.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2916 http.(*conn).serve at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:1966]"
```

The exception occurs when 
```
 service.passwordFromAuthenticatorInfo at /.../authgear-server/pkg/lib/authn/authenticator/service/conversion.go:34
```

Maybe the claim changed to string during serialization and deserialization? Better to use string in claims

**Review by carmenlau — COMMENTED** [review]
Need updating templates for ClaimKey. Got following error when visiting settings page, for the user who use otp to login.
```
ERRO[0230] panic occurred                                app=accounts error="failed to execute html template: template: web/settings.html:294:9: executing \"page-content\" at <index .Claims \"https://authgear.com/claims/oob_otp/email\">: error calling index: value has type string; should be authenticator.ClaimKey" logger=webapp-panic-middleware stack="[log.PanicValue at /Users/carmenlau/work/authgear/authgear-server/pkg/util/log/panic.go:56 webapp.(*PanicMiddleware).Handle.func1.3 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/panic_middleware.go:53 runtime.gopanic at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/runtime/panic.go:838 db.(*HookHandle).WithTx.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/infra/db/hook_handle.go:78 runtime.gopanic at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/runtime/panic.go:838 webapp.(*ResponseRenderer).RenderStatus at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/renderer.go:47 webapp.(*ResponseRenderer).Render at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/renderer.go:32 webapp.(*ResponseRenderer).RenderHTML at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/renderer.go:63 webapp.(*SettingsHandler).ServeHTTP.func2 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/settings.go:158 db.(*HookHandle).WithTx at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/infra/db/hook_handle.go:89 webapp.(*Controller).Serve at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/controller.go:143 webapp.(*SettingsHandler).ServeHTTP at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/settings.go:216 deps.(*RootProvider).Handler.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:170 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.RequireAuthenticatedMiddleware.Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/require_authenticated_middleware.go:28 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*DynamicCSPMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/dynamic_csp_middleware.go:49 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 web.StaticSecurityHeadersMiddleware.Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/web/static_security_headers_middleware.go:10 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 csrf.(*csrf).ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/packages/pkg/mod/github.com/gorilla/csrf@v1.7.1/csrf.go:307 webapp.(*CSRFMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/csrf_middleware.go:52 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*TutorialMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/tutorial_middleware.go:26 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*ClientIDMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/client_id.go:34 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*WeChatRedirectURIMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/wechat_redirect_uri_middleware.go:73 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*ColorSchemeMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/x_color_scheme.go:78 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*UILocalesMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/ui_locales.go:48 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*SessionMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/session_middleware.go:25 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.IntlMiddleware.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/intl_middleware.go:14 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 httputil.NoCache.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/util/httputil/no_cache.go:12 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 session.(*Middleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/session/middleware.go:64 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*PanicMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/handler/webapp/panic_middleware.go:64 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 webapp.(*PublicOriginMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/auth/webapp/public_origin_middleware.go:36 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RootProvider).Middleware.func1.1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/providers.go:184 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 deps.(*RequestMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/deps/middleware_request.go:39 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 middleware.(*SentryMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/infra/middleware/sentry.go:22 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 middleware.(*BodyLimitMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/infra/middleware/body_limit.go:16 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 middleware.(*PanicMiddleware).Handle.func1 at /Users/carmenlau/work/authgear/authgear-server/pkg/lib/infra/middleware/panic.go:63 http.HandlerFunc.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2084 httprouter.(*Router).Handler.func1 at /Users/carmenlau/.asdf/installs/golang/1.18.2/packages/pkg/mod/github.com/julienschmidt/httprouter@v1.3.0/router.go:275 httprouter.(*Router).ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/packages/pkg/mod/github.com/julienschmidt/httprouter@v1.3.0/router.go:387 httproute.(*Router).ServeHTTP at /Users/carmenlau/work/authgear/authgear-server/pkg/util/httproute/httproute.go:82 http.serverHandler.ServeHTTP at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:2916 http.(*conn).serve at /Users/carmenlau/.asdf/installs/golang/1.18.2/go/src/net/http/server.go:1966]"
```

The other parts look good!

File: pkg/lib/authn/authenticator/service/conversion.go (hunk 1 above)
Comment (louischan-oursky) [reply pkg/lib/authn/authenticator/service/conversion.go:34]: I cannot reproduce this issue.

Comment (carmenlau) [reply pkg/lib/authn/authenticator/service/conversion.go:34]: I tried to signup with email otp + additional password, and failed in the setup recovery page. See if you can reproduce it.

**Pull request merged**